artifacts/camara-cloudevents/event-subscription-template.yaml

```
@@ -0,0 +1,951 @@
+openapi: 3.0.3
+info:
+  title: Notification Subscription Template
+  description: |
+    This file is a template for CAMARA API explicit subscription endpoint and for Notification model. Additional information are provided in API Design Guidelines document.
+
+    Note on ``security`` - ``openId`` scope: The value in this yaml `api-name:event-type:grant-level` must be replaced accordingly to the format defined in the guideline document.
+
+  termsOfService: http://swagger.io/terms/
+  contact:
+    email: project-email@sample.com
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.1.0-wip
+externalDocs:
+  description: Product documentation at CAMARA
+  url: https://github.com/camaraproject/
+servers:
+  - url: "{apiRoot}/{apiName}/v0"
+    variables:
+      apiRoot:
+        default: http://localhost:9091
+        description: API root
+      apiName:
+        default: camaraAPI
+        description: apiName will be replaced in WG by its value and not modelled as a variable
+tags:
+  - name: apiName subscription
+    description: Operations to manage event subscriptions on event-type event
+
+paths:
+  /subscriptions:
+    post:
+      tags:
+        - apiName subscription
```

For tags we should use Title Case

```suggestion
        - <apiName> Subscription
```